tests/r2u.sh

```diff
@@ -14,3 +16,10 @@ install.r BiocParallel
 echo "TEST: install binary deps + binary from r-universe"
 REPO_URL="https://eddelbuettel.r-universe.dev/bin/linux/$VERSION_CODENAME/${RVER%.*}"
 install2.r -r $REPO_URL RcppKalman
+
+# Tests for default path
+echo "options(bspm.version.check=TRUE)" >> /etc/R/Rprofile.site
+echo "options(install.packages.compile.from.source='never')" >> /etc/R/Rprofile.site
```

Comment:
Is that one new?

Reply:
No, it's the same. But the failure was specific to this code path, so I added this section.